framework/src/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -505,6 +589,15 @@ public AnnotationMirror leastUpperBound(AnnotationMirror a1, AnnotationMirror a2
                     newValues.addAll(a1Values);
                     newValues.addAll(a2Values);
                     return createArrayLenAnnotation(newValues);
+                } else if (AnnotationUtils.areSameByClass(a1, StringVal.class)) {
+                    List<String> a1Values =
+                            AnnotationUtils.getElementValueArray(a1, "value", String.class, true);
```

Comment:
@smillst, there are also similar calls not written by me. Should I change them too or just the new ones?